internal/provider/apikey_resource.go

```diff
@@ -183,10 +185,15 @@ func (r *apiKeyResource) Create(ctx context.Context, req resource.CreateRequest,
 		disabled = plan.Disabled.ValueBool()
 	}
 
+	ownerType, err := enums.ToOwnerType(plan.OwnerType.ValueString())
```

Comment:
From a String perspective, what is the delta between the old version's owner type and the Enums owner type? 
For context, asking to understand the impact to existing state for TF users who currently manage API Key resources

Reply:
confirmed this behavior locally.  TY